configuration/go/sdk/config-subscriber/app.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	dapr "github.com/dapr/go-sdk/client"
+)
+
+func main() {
+	client, err := dapr.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	CONFIGURATION_STORE := "configstore"
```

Comment:
Any reason this variable name is in ALL CAPS? Is this intended to be a global variable? Please fix the case if otherwise.